### DISCUSSION TITLE
How to use Grid in a custom page view

### DISCUSSION THREAD

**Discussion by realmocaccino** [filamentphp/filament#13904] [Help] [answered]
### Package

Other

### Package Version

v3.2.95

### How can we help you?

Hi guys!
I want to take advantage of the Filament's Grid system , but I don't know where to start given I am not using a Panel, Form or Table builder.
I have a custom page class with a view property set, where this same view file calls methods from that class.
How would I use the Grid in that view file?
Thank you.

**Comment by leandrocfe** [comment, marked as answer]
You can use this grid https://filamentphp.com/docs/3.x/forms/layout/grid if you are using a Form Builder in a livewire component: https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component#adding-the-form

If you want to use the blade component, you could do something like that

```html
<x-filament::grid
    style="--cols-lg: repeat(3, minmax(0, 1fr));"
    class="lg:grid-cols-[--cols-lg]"
>
    <div>1</div>
    <div>2</div>
    <div>3</div>

</x-filament::grid>
```

`--cols-lg`: breakpoint (sm, md, lg...)
`repeat(3`: 3 columns

**Comment by realmocaccino** [comment]
Thank you man!
